cmake/modules/SwiftUtils.cmake

```diff
@@ -152,6 +152,43 @@ function(swift_create_post_build_symlink target)
     COMMENT "${CS_COMMENT}")
 endfunction()
 
+function(swift_create_early_driver_symlinks target)
+  # Is there a better way to get the top-level `build` dir prefix?
+  set(driver_bin_dir "${CMAKE_CURRENT_BINARY_DIR}/../../../earlyswiftdriver-${SWIFT_HOST_VARIANT}-${SWIFT_HOST_VARIANT_ARCH}/release/bin")
```

Comment:
This should be passed into CMake on the command line.

Reply:
Used CMake's existing specification of the `swift` build directory that was passed into it on the command line instead.